com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.x.x]
+### Fixed
+- Fixed an issue where 2D Renderer was generating garbage in editor.
+
```

Comment:
Shouldn't this be added to 7.8 instead? it seems like 7.8 is current development? In that case you should remove the "version updated" text and add this category.

Reply:
Fixed